packages/design-system-react/src/components/button-base/ButtonBase.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { ButtonBase } from './ButtonBase';
+import { ButtonBaseSize } from './ButtonBase.types';
+import README from './README.mdx';
+
+const meta: Meta<typeof ButtonBase> = {
+  title: 'React Components/ButtonBase',
+  component: ButtonBase,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    size: {
+      control: 'select',
+      options: Object.values(ButtonBaseSize),
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ButtonBase>;
+
+export const Default: Story = {
+  args: {
+    children: 'Button Base',
+  },
+};
+
+export const Size: Story = {
+  render: () => (
+    <div className="flex gap-2">
+      <ButtonBase size={ButtonBaseSize.Sm}>Small</ButtonBase>
+      <ButtonBase size={ButtonBaseSize.Md}>Medium</ButtonBase>
+      <ButtonBase size={ButtonBaseSize.Lg}>Large</ButtonBase>
+    </div>
+  ),
+};
+
+export const FullWidth: Story = {
```

Comment:
Make sure the stories and titles in readme are a direct match to the prop name just in PascalCase